packages/fiori/src/UserMenuAccount.ts

```diff
@@ -0,0 +1,84 @@
+import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
+import { customElement, property } from "@ui5/webcomponents-base/dist/decorators.js";
+
+@customElement({
+	tag: "ui5-user-menu-account",
+})
+/**
+ * @class
+ * ### Overview
+ *
+ * The `ui5-user-menu-account` represents an account in the `ui5-user-menu`.
+ *
+ * ### ES6 Module Import
+ * `import "@ui5/webcomponents-fiori/dist/UserMenuAccount.js";`
+ *
+ * @constructor
+ * @extends UI5Element
+ * @experimental
+ * @public
+ * @since 2.5.0
+ */
+class UserMenuAccount extends UI5Element {
+	/**
+	 * Defines the avatar of the user.
+	 *
+	 * @default ""
+	 * @public
+	 */
+	@property({ type: String })
+	avatar?: string;
+
+	/**
+	 * Defines the avatar initials of the user.
+	 *
+	 * @default undefined
+	 * @public
+	 */
+	@property({ type: String })
+	initials?: string;
+
+	/**
+	 * Defines the title text of the user.
+	 *
+	 * @default ""
+	 * @public
+	 */
+	@property({ type: String })
+	titleText = "";
+
+	/**
+	 * Defines additional text of the user.
+	 *
+	 * @default ""
+	 * @public
+	 */
+	@property({ type: String })
+	additionalText = "";
```

Comment:
We agreed on the following props:
- **avatarSrc** (rename avatar)
- **avatarInitials** (rename initials)
- titleText (remains)
- **subTitleText** (rename additionalText)
- description (remains)
- selected (remains)